metals/src/main/resources/db/migration/V4__Add_sbt_digest.sql

```diff
@@ -0,0 +1,8 @@
+drop table if exists sbt_fingerprint_event;
+drop table if exists sbt_checksum;
```

Comment:
I wrote it mostly for myself because it's annoying to manually clean the db for the projects I'm testing it with. I changed the migration handling to fallback to cleaning the db in case of migration exceptions. So far there is no critical information in the db, it's OK to reset it.